scraping/scraping/data/hm_fr_male_female.json

```diff
@@ -1,641 +1,728 @@
             "FEMALE",
             "MALE"
         ],
-        "category": "TSHIRT"
-    },
-    "top": {
-        "name": "Top",
-        "code": "Top",
-        "gender": [
-            "FEMALE"
-        ],
-        "category": "TOP"
+        "filters": {
+            "productTypes": "Legging"
+        }
     },
-    "troussedetoilette": {
-        "name": "Trousse de toilette",
-        "code": "Trousse+de+toilette",
+    {
+        "category": "PANTS",
         "gender": [
-            "FEMALE"
+            "FEMALE",
+            "MALE"
         ],
-        "category": ""
+        "filters": {
+            "productTypes": "Pantalon"
+        }
     },
-    "tunique": {
-        "name": "Tunique",
-        "code": "Tunique",
+    {
+        "category": "PANTS",
         "gender": [
-            "FEMALE"
+            "FEMALE",
+            "MALE"
         ],
-        "category": ""
+        "filters": {
+            "productTypes": "Short"
+        }
     },
-    "veste": {
-        "name": "Veste",
-        "code": "Veste",
+    {
+        "category": "SHIRT",
         "gender": [
             "FEMALE",
             "MALE"
         ],
-        "category": "JACKET"
+        "filters": {
+            "productTypes": "Chemise"
+        }
     },
-    "vestesansmanches": {
-        "name": "Veste sans manches",
-        "code": "Veste+sans+manches",
+    {
+        "category": "SHOES",
         "gender": [
             "FEMALE",
             "MALE"
         ],
-        "category": "JACKET"
+        "filters": {
+            "productTypes": "Chaussons"
+        }
     },
-    "vêtementspourchien": {
-        "name": "Vêtements pour chien",
-        "code": "V%C3%AAtements+pour+chien",
+    {
+        "category": "SHOES",
         "gender": [
             "FEMALE",
             "MALE"
         ],
-        "category": ""
+        "filters": {
+            "productTypes": "Chaussures"
+        }
     },
-    "étuiàlingettes": {
-        "name": "Étui à lingettes",
-        "code": "%C3%89tui+%C3%A0+lingettes",
+    {
+        "category": "SKIRT",
         "gender": [
             "FEMALE"
         ],
-        "category": ""
+        "filters": {
+            "productTypes": "Jupe"
+        }
     },
-    "caleçonlong": {
-        "name": "Caleçon long",
-        "code": "Cale%C3%A7on+long",
+    {
+        "category": "SNEAKERS",
         "gender": [
+            "FEMALE",
             "MALE"
         ],
-        "category": ""
+        "filters": {
+            "productTypes": "Chaussures",
+            "footwearStyles": "Baskets%2Ftennis"
+        }
     },
-    "cravate": {
-        "name": "Cravate",
-        "code": "Cravate",
+    {
+        "category": "SOCKS",
         "gender": [
+            "FEMALE",
             "MALE"
         ],
-        "category": ""
+        "filters": {
+            "productTypes": "Chaussettes"
+        }
     },
-    "gants": {
-        "name": "Gants",
-        "code": "Gants",
+    {
+        "category": "SUIT",
```

Comment:
Initially, we had `SUIT`, but you assigned in all the other shops `Blazer` to `JACKET`. And this is also GS1 GPC conform. So we should do it here as well.